app/response_processor.py

```diff
@@ -125,7 +127,7 @@ def store_survey(self, decrypted_json):
             raise QuarantinableError
 
         self.logger.info("Survey storage successful")
-        return response
+        return response.json()
```

Comment:
If we add the additional endpoint in sdx-store to add a feedback response, then in this function you'll need to do an if statement to determine if the decrypted data is a feedback submission and if so route to the new endpoint

Reply:
as discussed this comment can be ignored now